src/components/SearchBar/MobileSearchBar.jsx

```diff
@@ -22,7 +22,8 @@ function MobileSearchBar({
   onChangeByLocation,
   onChangeByStatus,
   onChangeByTitle,
-  onSearch,
+  //onSearch,
+  onClick,
```

Comment:
I noticed you renamed the prop from `onSearch` to `onClick`. Could you explain the reasoning behind this change?

I initially named the prop `onSearch` to describe its intended use more clearly. The MobileSearchBar component has two clickable elements - one to open the filter modal and another to trigger the search. I'm concerned that onClick might be too generic and could lead to confusion about which action it's tied to.

Was there a specific reason for choosing onClick? Let me know your thoughts!

Reply:
`onClick` is passed as a prop from `DesktopSearchBar` so renamed it to maintain uniformity. I can change it back to `onSearch`.

Reply:
Ah gotcha, understood! Thanks for pointing that out. 

Can you please change all the instances of `onClick` to `onSearch` in the`SearchBar`, `DesktopSearchBar`, and `MobileSearchBar` components? Thank you!

Reply:
Done. Updated!